demoapps/simplest/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="org.openyolo.demoapp.simplest">
+
+    <application
+        android:allowBackup="true"
+        android:fullBackupContent="false"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+            <intent-filter>
```

Comment:
I think we can remove this intent filter. Copy pasta from demoapps/passwordlogin/AndroidManifest.xml :)? if this is removed we should do the same there too.

Reply:
This is mandatory by Lint, removing it will cause a warning (so an error…)

Reply:
Really? The <data> tag sticks out to me unrelated, at the very least using an incorrect host value.

Reply:
Sorry, I just re-checked, and only the intent-filter VIEW is necessary, not the website. I will remove it.